src/js/elements/mapchip/legend.js

```diff
@@ -0,0 +1,51 @@
+import {format as d3format} from 'd3-format';
+
+import { Observable } from "../../utils";
+
+const lightStart = 3;
```

Comment:
might want to pass this into the constructor

Reply:
done